src/elements/atoms/Divider/Divider.jsx

```diff
@@ -0,0 +1,69 @@
+/** Divider element is a dedicated component to give visual spacing between componentry. */
+
+export class Divider extends React.Component {
+  static propTypes = {
+    /** Tag overload */
+    tagName: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element,
+      PropTypes.func
+    ]),
+    /** Class stacking */
+    className: PropTypes.string,
+    /** Style variants */
+    variant: PropTypes.oneOf(['horizontal', 'vertical']),
+    /** Defines how wide it is across the screen */
+    length: PropTypes.oneOf(['large', 'medium', 'medium-small', 'small']),
+    /** Defines the thickness of the divider */
+    thickness: PropTypes.oneOf(['large', 'medium', 'small']),
+    /** Defines the color of the line */
+    color: PropTypes.oneOf(['black', 'gray', 'red', 'green']),
+    /** Children passed through */
+    children: PropTypes.node
+  };
+
+  static defaultProps = {
+    tagName: 'div',
+    variant: 'horizontal',
+    length: 'large',
+    thickness: 'medium',
+    color: 'black'
+  };
+
+  /** Element level options */
+  static options = {};
+
+  render = () => {
+    const {
+      tagName: Tag,
+      className,
+      variant,
+      children,
+      orientation,
+      length,
+      thickness,
+      color,
+      ...attrs
+    } = this.props;
+
+    const classStack = Utils.createClassStack([
+      'divider',
+      `divider--${variant}`,
+      `divider--color-${color}`,
+      `divider--length-${length}`,
+      `divider--thickness-${thickness}`,
+      className
+    ]);
+
+    return (
+      <Tag
+        className={classStack}
+        {...attrs}
```

Comment:
Lets add a role="presentation".

"An element whose implicit native role semantics will not be mapped to the accessibility API."

I guess there use to be a `role="seperator"` that has been depreciated or recommended to no longer use.